module/zstd/zstd.c

```diff
@@ -253,9 +253,6 @@ zstd_mempool_alloc(struct zstd_pool *zstd_mempool, size_t size)
 			mem->kmem_type = ZSTD_KMEM_DEFAULT;
 			mem->kmem_size = size;
 		}
-	} else {
```

Comment:
huh? where is that set?

Reply:
eh... just forget it....

Reply:
direct at allocation. should never change. so no need to reset it

Reply:
yeah, I just realized that right after my comment :)